### PR TITLE
[FR] Update Beatmap Editor/Beat Snap Divisor

### PR DESCRIPTION
**Pull request by DoryanR**
Translation of the Beat Snap Divisor page

### CODE REVIEW

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 1]
```diff
@@ -1,16 +1,121 @@
-# Représentation du Tempo
+# Beat Snap Diviseur
```
Comment (almaloste) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]:
```suggestion
# Beat Snap Divisor
```

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 2]
```diff
@@ -1,16 +1,121 @@
...
 
-Le diviseur de l’alignement des battements est utilisé pour limiter l'endroit où les notes peuvent être placées, de sorte qu'elles soient toujours sur un battement. Déplacer le slider sur la droite augmente la granularité des notes, ie. plus de notes peuvent être placées sur la même mesure.
+Le Beat Snap Diviseur est utilisé pour limiter l'endroit où les notes peuvent être placées afin qu'elles tombent toujours sur le temps. En déplaçant le curseur vers la droite, on augmente la granularité des notes, c'est-à-dire qu'on peut placer plus de notes dans la même mesure.
```
Comment (almaloste) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]:
```suggestion
Le Beat Snap Divisor est utilisé pour limiter l'endroit où les notes peuvent être placées afin qu'elles tombent toujours sur le temps. En déplaçant le curseur vers la droite, on augmente la granularité des notes, c'est-à-dire qu'on peut placer plus de notes dans la même mesure.
```

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 3]
```diff
@@ -1,16 +1,121 @@
...
 
-Chacune de ces configurations ajoute des couleurs différentes de tics sur la [ligne du temps](/wiki/Beatmap_Editor/Timelines) . Les tics sont comme il suit :
+Chacun des réglages met différentes couleurs de ticks sur le [Timelines](/wiki/Beatmap_Editor/Timelines). Les cotes sont les suivantes (pour [Standard](/wiki/Game_Modes/osu!), [osu!taiko](/wiki/Game_Modes/osu!taiko), et [osu!catch](/wiki/Game_Modes/osu!catch)):
```
Comment (almaloste) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]:
```suggestion
Chacun des réglages met différentes couleurs de ticks sur les [timelines](/wiki/Beatmap_Editor/Timelines). Les ticks sont les suivantes (pour [Standard](/wiki/Game_Modes/osu!), [osu!taiko](/wiki/Game_Modes/osu!taiko), et [osu!catch](/wiki/Game_Modes/osu!catch)):
```

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 4]
```diff
@@ -1,16 +1,121 @@
...
-- 1/1 = Tic blanc seulement
-- 1/2 = Ajoute un tic rouge
-- 1/3 = Ajoute deux tics violets entre chaque tic blanc
-- 1/4 = Ajoute deux tics bleu en plus du tic rouge
-- 1/6 = Ajoute deux tics violets entre chaque tic rouge
-- 1/8 = Ajoute un tic jaune en plus du tic rouge et bleu
+### Diviseur 1/1
 
-Le 1/3 et 1/6 sont particuliers, car cela signifie que vous mappez sur une map au rythme ternaire.
+![Divisor 1/1](/wiki/shared/BSD_1_1b.jpg "Diviseur 1/1")
```
Comment (almaloste) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]:
```suggestion
![Diviseur 1/1](/wiki/shared/BSD_1_1b.jpg "Diviseur 1/1")
```

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 5]
```diff
@@ -1,16 +1,121 @@
...
 
-Maintenir la touche alt au-dessus d'un slider modifie la [distance d'alignement](/wiki/Beatmap_Editor/Distance_Snap) du slider
+- Battements complets
+- Ticks blanches seulement
+- A utiliser de préférence pour la difficulté facile.
+
+### Diviseur 1/2
+
+![Divisor 1/2](/wiki/shared/BSD_1_2.jpg "Diviseur 1/2")
```
Comment (almaloste) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]:
```suggestion
![Diviseur 1/2](/wiki/shared/BSD_1_2.jpg "Diviseur 1/2")
```

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 6]
```diff
@@ -1,16 +1,121 @@
...
+
+- Demi-temps
+- Ajoute des ticks rouges
+
+### Diviseur 1/3
+
+![Divisor 1/3](/wiki/shared/BSD_1_3.jpg "Diviseur 1/3")
```
Comment (almaloste) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]:
```suggestion
![Diviseur 1/3](/wiki/shared/BSD_1_3.jpg "Diviseur 1/3")
```

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 7]
```diff
@@ -1,16 +1,121 @@
...
+
+- Triplets *
+- A deux ticks violettes entre chaque tique blanche
+- Timing composé
+
+### Diviseur 1/4
+
+![Divisor 1/4](/wiki/shared/BSD_1_4.jpg "Diviseur 1/4")
```
Comment (almaloste) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]:
```suggestion
![Diviseur 1/4](/wiki/shared/BSD_1_4.jpg "Diviseur 1/4")
```

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 8]
```diff
@@ -1,16 +1,121 @@
...
+
+- Quarts de temps
+- Ajoute les ticks bleues, en plus des ticks rouges
+- Utile pour la cartographie générale.
```
Comment (almaloste) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]:
```suggestion
- Utile pour le mapping.
```

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 9]
```diff
@@ -1,16 +1,121 @@
...
+
+### Diviseur 1/6
+
+![Divisor 1/6](/wiki/shared/BSD_1_6.jpg "Diviseur 1/6")
```
Comment (almaloste) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]:
```suggestion
![Diviseur 1/6](/wiki/shared/BSD_1_6.jpg "Diviseur 1/6")
```

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 10]
```diff
@@ -1,16 +1,121 @@
...
+
+Double Triplets*
+- A deux ticks violettes entre chaque ticks rouge
+- Timing composé
+
+### Diviseur 1/8
+
+![Divisor 1/8](/wiki/shared/BSD_1_8.jpg "Diviseur 1/8")
```
Comment (almaloste) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]:
```suggestion
![Diviseur 1/8](/wiki/shared/BSD_1_8.jpg "Diviseur 1/8")
```

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 11]
```diff
@@ -1,16 +1,121 @@
...
+
+- Huitième temps
+- Ajoute des ticks jaunes, en plus des ticks rouges et bleues
+
+Les 1/3 et 1/6 sont spéciaux car cela signifie que votre BPM est divisible par trois, mais c'est *rare*.
+
+En maintenant enfoncée la touche `Alt` sur le slider le change l'[espacement des distances](/wiki/Beatmap_Editor/Distance_Snap) du slider
```
Comment (almaloste) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]:
```suggestion
En maintenant enfoncée la touche `Alt` sur le changement de slider l'[espacement des distances](/wiki/Beatmap_Editor/Distance_Snap) du slider
```

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 12]
```diff
@@ -1,16 +1,121 @@
...
+
+Double ligne blanche : Mesure
+
+Ligne blanche : Temps commun
+
+Ligne verte : Vous.
+
+### Diviseur 1/1
+
+![Divisor 1/1](/wiki/shared/1_1_m.jpg "Diviseur 1/1")
```
Comment (almaloste) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]:
```suggestion
![Diviseur 1/1](/wiki/shared/1_1_m.jpg "Diviseur 1/1")
```

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 13]
```diff
@@ -1,16 +1,121 @@
...
+
+- Battements complets
+- Ticks blanches seulement
+- A utiliser de préférence pour la difficulté facile.
+
+### Diviseur 1/2
+
+![Divisor 1/2](/wiki/shared/1_2_m.jpg "Diviseur 1/2")
```
Comment (almaloste) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]:
```suggestion
![Diviseur 1/2](/wiki/shared/1_2_m.jpg "Diviseur 1/2")
```

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 14]
```diff
@@ -1,16 +1,121 @@
...
+
+- Demi-temps
+- Ajoute des ticks rouges
+
+### Diviseur 1/3
+
+![Divisor 1/3](/wiki/shared/1_3_m.jpg "Diviseur 1/3")
```
Comment (almaloste) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]:
```suggestion
![Diviseur 1/3](/wiki/shared/1_3_m.jpg "Diviseur 1/3")
```

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 15]
```diff
@@ -1,16 +1,121 @@
...
+
+- Triplets
+- A deux ticks violettes entre chaque ticks blanche
+- Chronométrage composé
+
+### Diviseur 1/4
+
+![Divisor 1/4](/wiki/shared/1_4_m.jpg "Diviseur 1/4")
```
Comment (almaloste) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]:
```suggestion
![Diviseur 1/4](/wiki/shared/1_4_m.jpg "Diviseur 1/4")
```

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 16]
```diff
@@ -1,16 +1,121 @@
...
+
+- Quarts de temps
+- Ajoute les ticks bleues, en plus des ticks rouges
+- Utile pour la cartographie générale
```
Comment (almaloste) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]:
```suggestion
- Utile pour le mapping
```

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 17]
```diff
@@ -1,16 +1,121 @@
...
+
+### Diviseur 1/6
+
+![Divisor 1/6](/wiki/shared/1_6_m.jpg "Diviseur 1/6")
```
Comment (almaloste) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]:
```suggestion
![Diviseur 1/6](/wiki/shared/1_6_m.jpg "Diviseur 1/6")
```

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 18]
```diff
@@ -1,16 +1,121 @@
...
+
+- Double Triplets
+- A deux ticks violettes entre chaque ticks rouge
+- Timing composé
+
+### Diviseur 1/8
+
+![Divisor 1/8](/wiki/shared/1_8_m.jpg "Diviseur 1/8")
```
Comment (almaloste) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]:
```suggestion
![Diviseur 1/8](/wiki/shared/1_8_m.jpg "Diviseur 1/8")
```

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 19]
```diff
@@ -1,16 +1,121 @@
...
+
+- Huitième temps
+- Ajoute des ticks jaunes, en plus des ticks rouges et bleues
+
+### Diviseur 1/12
+
+![Divisor 1/12](/wiki/shared/1_12_m.jpg "Diviseur 1/12")
```
Comment (almaloste) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]:
```suggestion
![Diviseur 1/12](/wiki/shared/1_12_m.jpg "Diviseur 1/12")
```

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 20]
```diff
@@ -1,16 +1,121 @@
...
+
+- Douze temps
+
+### Diviseur 1/16
+
+![Divisor 1/16](/wiki/shared/1_16_m.jpg "Diviseur 1/16")
```
Comment (almaloste) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]:
```suggestion
![Diviseur 1/16](/wiki/shared/1_16_m.jpg "Diviseur 1/16")
```

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 21]
```diff
@@ -1,16 +1,121 @@
...
+
+![Diviseur 1/12](/wiki/shared/1_12_m.jpg "Diviseur 1/12")
+
+- Douze temps
+
+### Diviseur 1/16
+
+![Diviseur 1/16](/wiki/shared/1_16_m.jpg "Diviseur 1/16")
+
+- Seizième temps
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]: please add an blank line after this line.

File: wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md [hunk 22]
```diff
@@ -1,16 +1,121 @@
...
+
+- Quarts de temps
+- Ajoute les ticks bleues, en plus des ticks rouges
+- Utile pour le mapping.
+
+### Diviseur 1/6
+
+![Diviseur 1/6](/wiki/shared/BSD_1_6.jpg "Diviseur 1/6")
+
+Double Triplets*
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor/Beat_Snap_Divisor/fr.md]:
```suggestion
- Double Triplets *
```
?

**Pull request merged**